scripts/kubernetes/sth-pod.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: v1
+kind: Pod
```

Comment:
why not `Deployment` ?

Reply:
I've changed this manifest to `kind: Deployment`